### ISSUE TITLE
Export contract terms

### ISSUE THREAD

**Issue by cams-goussale** [blockframes/blockframes#8908]
@Jackseed following your conversation with Bruce, I think it'd be necessary at some point to be able to export all terms from all contracts. Similar to what we have in the CRM with events: 1 list for Events, 1 list for Invitations, maybe we could have 1 list for Contracts, 1 list for Terms.

I don't think it's needed for the release (having the contracts export is enough for our kpis) but at some point if we need to audit our avails, this will be necessary.

**Comment by Jackseed** [comment]
Yep, lgtm but best to have 2 different exports as you advice instead of mixing terms in the contract export
